data_steward/utils/slack_alerts.py

```diff
@@ -22,7 +25,7 @@ class SlackConfigurationError(RuntimeError):
     """
 
     def __init__(self, msg):
-        super(SlackConfigurationError, self).__init__()
+        super(SlackConfigurationError, self).__init__(msg)
```

Comment:
I added this because it wasn't printing what the actual error was. Before when it failed it would just say:  
```raise SlackConfigurationError(UNSET_SLACK_TOKEN_MSG)```
I had to look through the actual code to see what that error was. When I added `msg`, it would say: 
```raise SlackConfigurationError(UNSET_SLACK_TOKEN_MSG)
utils.slack_alerts.SlackConfigurationError: Slack token not set in environment variable SLACK_TOKEN```

Is there a way to show this without adding `msg`?